### DISCUSSION TITLE
Use of full size raspberry pis.

### DISCUSSION THREAD

**Discussion by philiptdrew** [tjboldt/Apple2-IO-RPi#112] [General]
Getting hold of Zero 2 W is rather hard at the moment.  I was going to use a 3B+ using a ribbon cable.

Clearly the apple II bus cannot power this so I'll need to use an external power supply and set the jumper on the rev 6 board.

If anyone else has done this, do you have any advice, hits, tips or caveats that might be useful before I start?  Do you have pictures of your setup?

I'd like to learn from any prior experience out there.

Thank you.

**Comment by tjboldt** [comment]
I tested with external power before so what you are describing should work fine. Some notes though. Don't ever connect/disconnect the cable or insert/remove the card when there is any power (Apple II or external USB). The card itself requires 3.3V to power the chips from the output of the RPi so when the RPi isn't powered, the card will hang on boot instead of showing a proper message.

**Comment by philiptdrew** [comment]
So, insert and remove the card only when BOTH computers are powered down.  Makes sense.  Once installed it sounds like the correct power on order is PI (wait for boot to complete), then power the IIe.

Super, I look forward to receiving the Apple2-IO-RPi card.

**Comment by philiptdrew** [comment]
Got it, thank you.  What a snazzy purple color it is :)  I made a point to disconnect the power jumper immediately so I don't forget.

Once I have it hooked up and working I'll take some pictures.

**Comment by philiptdrew** [comment]
Out of curiosity I've ordered a banana pi BPi-M2 Pro that has similar performance to a 2GB Pi 4:

![image](https://user-images.githubusercontent.com/114966366/197224004-93e86069-1e29-48cf-a064-c2c761bee4ca.png)
https://wiki.banana-pi.org/Banana_Pi_BPI-M2_Pro

The form factor would seem to fit (though they also do a smaller Zero 2 W type device too.

I like the full size ethernet, built in wifi etc.  Quad core A55 & 2GB ram, overkill possibly but I think it'd work with the official Debian install.  I'll try the Pi 3 externally first, then I'll try this hooked directly to the Apple2-IO-RPi (but powered externally).

**Comment by oliverschmidt** [comment]
Using the recent board type (having the power jumper) there's no functional problem with directly attaching a full size Raspberry Pi - just don't try to attach the lid ;-)

![Back](https://user-images.githubusercontent.com/2664009/215295865-bf3fca98-4e9f-4e6d-9045-5a395fc355e4.jpg)

Both neighbor slots stay fully usable in that configuration.

![Bottom](https://user-images.githubusercontent.com/2664009/215295887-e1b175f5-6f2e-4785-8823-893ee28669c8.jpg)
